src/components/calcite-action-menu/calcite-action-menu.tsx

```diff
@@ -363,15 +363,7 @@ export class CalciteActionMenu {
   getActions = (): void => {
     const { el } = this;
 
-    const assignedActions = this.getAssignedElements().filter(
-      (element) => element.tagName === "CALCITE-ACTION" && element.slot !== "trigger"
-    ) as HTMLCalciteActionElement[];
-
-    const actionElements = assignedActions.length
-      ? assignedActions
-      : (Array.from(
-          el.querySelectorAll("calcite-action:not([slot=trigger])")
-        ) as HTMLCalciteActionElement[]);
+    const actionElements = getSlotted(el, { all: true, matches: "calcite-action" }) as any;
```

Comment:
Need to test this cleanup...

Reply:
I think this issue needs to happen first.... https://github.com/Esri/calcite-components/issues/3881

Reply:
> Test successful?

Yes this seems to be working.